RELEASING.rst

```diff
@@ -168,4 +168,8 @@ Both automatic and manual processes described above follow the same steps from t
    * python-announce-list@python.org (all releases)
    * testing-in-python@lists.idyll.org (only major/minor releases)
 
-   And announce it on `Twitter <https://twitter.com/>`_ and `BlueSky <https://bsky.app/>`_ with the ``#pytest`` hashtag.
+   And announce it with the ``#pytest`` hashtag on:
+
+   * `Twitter/X <https://x.com/>`_
+   * `BlueSky <https://bsky.app/>`_
```

Comment:
The official spelling has a lowercase "s":
```suggestion
   * `Bluesky <https://bsky.app>`_
```
![Screenshot_2025-01-11-12-51-48-05_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/user-attachments/assets/dfbdfd22-8776-4e46-9765-cf97cb70aaaa)